src/pages/ProfilePage.js

```diff
@@ -200,15 +201,16 @@ function ProfilePage(props) {
                             ) : null}
                             {shouldShowLocalTime && <AutoUpdateTime timezone={timezone} />}
                         </View>
-                        {!isCurrentUser && Boolean(login) && (
-                            <MenuItem
-                                title={`${props.translate('common.message')}${displayName}`}
-                                icon={Expensicons.ChatBubble}
-                                onPress={() => Report.navigateToAndOpenReport([login])}
-                                wrapperStyle={styles.breakAll}
-                                shouldShowRightIcon
-                            />
-                        )}
+                        {!isCurrentUser &&
+                            !Session.isAnonymousUser(
```

Comment:
```suggestion
                            !Session.isAnonymousUser() && (
```

Reply:
I was about to post that this isn't working for me, I think this is why.

<img width="1020" alt="Screenshot 2023-06-21 at 22 28 54" src="https://github.com/Expensify/App/assets/9680864/80641088-889b-4164-9038-6bf7c74f3d57">